demo/dvc.lock

```diff
@@ -31,4 +31,4 @@ stages:
       nfiles: 2
     - path: training_metrics.json
       md5: 2a592caac70651d359b2331ff674beec
-      size: 69
```

Comment:
Accidently committed some `demo` changes! Not sure what's causing this file change though 🤔